netty/src/main/java/io/grpc/netty/Utils.java

```diff
@@ -89,6 +90,10 @@ class Utils {
       = new DefaultEventLoopGroupResource(1, "grpc-nio-boss-ELG", EventLoopGroupType.NIO);
   public static final Resource<EventLoopGroup> NIO_WORKER_EVENT_LOOP_GROUP
       = new DefaultEventLoopGroupResource(0, "grpc-nio-worker-ELG", EventLoopGroupType.NIO);
+  public static final Resource<EventLoopGroup> UDS_CHANNELS_EVENT_LOOP_GROUP =
+      Epoll.isAvailable() ? new DefaultEventLoopGroupResource(1, "UdsChannels",
```

Comment:
The code was going through pains to not depend on epoll. See `isEpollAvailable()` in this class.

That is sorta not necessary any more because the class files have been split from the .so's in Maven Central, although the dependencies are still a bit weird due to Maven vs Gradle issues, so it is probably best to continue using reflection since it is so easy. Maybe later we can remove the reflection everywhere.